### PR TITLE
Support completions for --serviceaccount of kubectl create clusterrolebinding

### PR DESCRIPTION
**Pull request by superbrothers**
**What this PR does / why we need it**:
This PR supports completion for --serviceaccount of kubectl create clusterrolebinding.
```
$ kubectl create clusterrolebinding --serviceaccount <tab>
default:default      kube-public:default  kube-system:default
```

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
Support completion for --serviceaccount of kubectl create clusterrolebinding
```

### CODE REVIEW

**Comment by shiywang** [comment]
could we also handle zsh well ? bash works well, but when I try on zsh
```console
➜  amd64 git:(serviceaccount-flag-completions) ./kubectl create clusterrolebinding --serviceaccount=kube-system:default __handle_flag:25: bad math expression: operand expected at end of string
                                               ./kubectl create clusterrolebinding --serviceaccount=kube-system:default
```
serviceaccount was completed but after [TAB] it throws error

**Comment by superbrothers** [comment]
Yes, we can handle zsh well. This error is due to other problem. https://github.com/kubernetes/kubectl/issues/42, https://github.com/kubernetes/kubernetes/pull/48553
Please check with the following command:
```
$ kubectl create clusterrolebinding --serviceaccount kube-system:default <tab>
```

**Comment by shiywang** [comment]
@superbrothers cool, I will also take a look at that pr

**Comment by superbrothers** [comment]
@shiywang @droot ping

**Comment by soltysh** [comment]
I guess this will have to land for 1.9. I'll add milestone accordingly.

/lgtm

File: pkg/kubectl/cmd/cmd.go [hunk 1]
```diff
@@ -159,6 +159,16 @@ __kubectl_require_pod_and_container()
     return 0
 }
 
+# Print serviceaccounts across all namespaces in the format "<namespace>:<name>"
+__kubectl_get_serviceaccounts_all_namespaces()
+{
+    local template kubectl_out
+    template="{{ range .items }}{{ .metadata.namespace }}:{{ .metadata.name }} {{ end }}"
+    if kubectl_out=$(kubectl get -o template --template="${template}" serviceaccount --all-namespaces 2>/dev/null); then
```
Comment (liggitt) [inline pkg/kubectl/cmd/cmd.go]: getting across all namespaces isn't something I'd expect to be part of an autocomplete... it's not consistently available and retrieves a very large amount of data on large clusters

**Pull request closed without merging**